terraform/shared/shared.tf.json.template.py

```diff
@@ -50,37 +50,10 @@ def paren(s: str) -> str:
                             '$.errorCode = "*UnauthorizedOperation"',
                             '$.errorCode = "AccessDenied*"',
                         ])),
-                        # Filtering on the errorCode alone catches too many false
-                        # positives, so we exclude those logs with the additional
-                        # conditions below.
-                        paren(' || '.join([
-                            '$.eventSource != "s3.amazonaws.com"',
-                            '$.eventName != "GetObject"',
-                            '$.userIdentity.invokedBy != "cloudfront.amazonaws.com"',
-                        ])),
-                        paren(' || '.join([
-                            '$.eventSource != "s3.amazonaws.com"',
-                            '$.eventName != "HeadBucket"',
-                            '$.userIdentity.invokedBy != "config.amazonaws.com"',
-                        ])),
-                        paren(' || '.join([
-                            '$.eventSource != "s3.amazonaws.com"',
-                            '$.eventName != "HeadObject"',
-                            '$.userIdentity.invokedBy != "cloudfront.amazonaws.com"',
-                        ])),
-                        paren(' || '.join([
-                            '$.eventSource != "inspector2.amazonaws.com"',
-                            '$.eventName != "DescribeOrganizationConfiguration"',
-                        ])),
-                        paren(' || '.join([
-                            '$.eventSource != "elasticloadbalancing.amazonaws.com"',
-                            '$.eventName != "DescribeTargetGroupAttributes"',
-                            '$.userIdentity.invokedBy != "inspector2.amazonaws.com"',
-                        ])),
-                        paren(' || '.join([
-                            '$.eventSource != "ec2.amazonaws.com"',
-                            '$.eventName != "CreateNetworkInterface"',
-                        ]))
+                        # Note: Contrary to CloudWatch queries, a metric filter
+                        # pattern such as "$.foo != 'bar'" will not match a
+                        # record that does not contain the field "foo", nor is
+                        # there a way to match absense of a field.
```

Comment:
I don't think there is much of a point to this comment since we're abandoning this approach entirely.